keps/sig-scheduling/20200114-multi-scheduling-profiles.md

```diff
@@ -192,6 +191,9 @@ similar result as the binary is starting.
 `SchedulerName` values will be validated to not repeat among the items of
 `Profiles`.
 
+Since kube-scheduler has only one queue, we will validate that all `Plugins.QueueSort`
```

Comment:
Let's list it as a test item (UT or integration test) explicitly.

Reply:
Added "validation" to the tests.

I think what the default sorting is isn't important for this KEP.